Description: Non-existent macro in manpage.
 Use an existing macro.
Author: Mats Erik Andersson <debian@gisladisker.se>
Forwarded: http://www.freebsd.org/cgi/query-pr.cgi?pr=154215
X-Patch-Applied: Fixed in HEAD by r205076.
Last-Update: 2011-01-21

--- freebsd-8.1.orig/sbin/quotacheck/quotacheck.8	2010-06-14 04:09:06.000000000 +0200
+++ freebsd-8.1/sbin/quotacheck/quotacheck.8	2011-01-21 14:22:47.000000000 +0100
@@ -82,7 +82,7 @@
 .Xr fsck 8 .
 This option is deprecated and parallel passes are always run
 as per
-.Xf fsck 8.
+.Xr fsck 8 .
 .It Fl u
 Only user quotas listed in
 .Pa /etc/fstab
